Containerfile

```diff
@@ -1,7 +1,7 @@
 FROM quay.io/centos/centos:stream9 AS builder
 
 RUN dnf install --enablerepo=crb -y \
-        clang-20.1.8-1.el9 \
+        clang-19.1.7 \
```

Comment:
clang version has been changed to match the one used downstream. Since this is an older version, it should provide better compatibility, so I think it is worth it.